cmd/goal/network.go

```diff
@@ -80,10 +88,23 @@ var networkCreateCmd = &cobra.Command{
 		if err != nil {
 			panic(err)
 		}
-		networkTemplateFile, err := filepath.Abs(networkTemplateFile)
-		if err != nil {
-			panic(err)
+
+		var defaultReader io.Reader
+
+		if networkTemplateFile == "" {
+			if devModeOverride {
+				defaultReader = strings.NewReader(defaultNetworkTemplate)
+			} else {
+				defaultReader = strings.NewReader(defaultDevNetworkTemplate)
+			}
+		}
```

Comment:
The dev mode override is intended to override after loading the template file. So hopefully you can skip this code and leverage the existing override mechanism.

Reply:
I was thinking we might want to implement some other differences for dev mode. Seems not.